### DISCUSSION TITLE
Seeing the original/translation as GM but players only seeing translation

### DISCUSSION THREAD

**Discussion by SEWFlyers** [mclemente/fvtt-module-polyglot#202] [Q&A] [answered]
Hello, I'm experimenting with Polyglot as it's a brilliant idea and would add a lot of fun to my upcoming game. However, I feel like I'm doing it backward and am not sure why. I'm using D&D5e. If I type in the phrase I want to appear to the players in Infernal ("Here Lies Lord Thalivar"), from the GM account I can see both the phrase in both "Infernal" and then in Common appearing right below it as "Translated from Infernal." However, from a player's account logged in on another browser for a character that doesn't speak Infernal, I'm only seeing the translation in Common and not the Infernal (which is the opposite of what I would expect). Is this a setting I need to change or something?

I've tried it from both the chat box as well as using a macro to script it (using the text from your other discussion thread). 

<img width="309" alt="Screen Shot 2021-11-13 at 8 09 52 PM" src="https://user-images.githubusercontent.com/51861395/141663741-1dcee720-344e-4a12-8eea-6e5c262a804a.png">

**Comment by mclemente** [comment]
Hey, could you zip your world's folder and send it over here? It'll be easier to track down what's causing it.

**Reply by SEWFlyers** [reply]
[worlds.zip](https://github.com/League-of-Foundry-Developers/fvtt-module-polyglot/files/7534305/worlds.zip)

**Comment by mclemente** [comment, marked as answer]
Go into the Language Settings and Reset Defaults. For some reason all the fonts got set to use the Common alphabet.

**Reply by SEWFlyers** [reply]
Hmm, you're right, but I have no idea why. Unfortunately, after I hit reset and also hit save settings, it's still doing the same thing. I see both Infernal and Translated as GM but only see the Translated version as a player who doesn't have Infernal as their language. It doesn't matter whether I have Common or Infernal selected in the chat window or if I'm using the macro. Here are my Polyglot settings in case I messed up something there. 
 
<img width="301" alt="Screen Shot 2021-11-14 at 4 05 39 PM" src="https://user-images.githubusercontent.com/51861395/141698651-469f1fe4-464e-4653-afe8-f1f3ab029902.png">
<img width="314" alt="Screen Shot 2021-11-14 at 4 05 48 PM" src="https://user-images.githubusercontent.com/51861395/141698652-936c973e-8fc1-48ab-a858-46d665701485.png">
<img width="597" alt="Screen Shot 2021-11-14 at 4 07 19 PM" src="https://user-images.githubusercontent.com/51861395/141698653-e031b94d-2a7f-49d3-b7ab-cf7f46ace236.png">

**Reply by mclemente** [reply]
Does that player has Owner/Observer permission on any characters that know Infernal?

**Reply by SEWFlyers** [reply]
Nope. As I'm moving to Foundry from a real-world setup, I haven't imported the other characters and only just set up this player. So in this case, it's a new player that I currently control that has ownership of only one actor (imported from D&D Beyond), and they only speak common, elvish, draconic, dwarvish, and goblin.

**Reply by SEWFlyers** [reply]
And when I say, "I control," it's actually a separate player account I set up that is different than my GM account, and I'm logging into it on a different browser to be able to test the "what does a player see" view. Sorry, I realized I might be confusing there.

**Reply by mclemente** [reply]
Does that player has a character in the map? Otherwise they're speaking OOC and the setting is set for GM Only.

**Comment by SEWFlyers** [comment]
Sorry mclemente, just getting back to messing around with this. Yes, the player is on the map, and I've experimented with them being visible and invisible, but it didn't affect anything. I also changed the setting from GM Only to PCs Only as well as All, as well as having a specific NPC speak in Infernal and other languages, and I'm still seeing the translated version for my PC (who doesn't speak the language). 🤷‍♂️
